gapic-generator/lib/gapic/presenters/method_rest_presenter.rb

```diff
@@ -37,7 +37,10 @@ class MethodRestPresenter
       def initialize main_method, api
         @main_method = main_method
         @http_bindings = main_method.http_bindings
-        @pagination = Gapic::Presenters::Method::RestPaginationInfo.new main_method.method, api
+        @compute_pagination =
+          if @main_method.service.special_compute_behavior?
+            Gapic::Presenters::Method::RestPaginationInfo.new main_method.method, api
```

Comment:
If we're going with "special" and "compute" everywhere do you want to rename RestPaginationInfo?

Reply:
I wasn't going to bother. In a future refactor we might want to have a "pagination info" class/subclass for the AIP algorithm as well, and we can rename the classes at that time to distinguish them.

Reply:
sgtm